audio/src/lewton_decoder.rs

```diff
@@ -17,29 +17,6 @@ where
     pub fn new(input: R) -> Result<VorbisDecoder<R>, VorbisError> {
         Ok(VorbisDecoder(OggStreamReader::new(input)?))
     }
-
-    pub fn seek(&mut self, ms: i64) -> Result<(), VorbisError> {
-        let absgp = ms * 44100 / 1000;
-        self.0.seek_absgp_pg(absgp as u64)?;
-        Ok(())
-    }
-
-    pub fn next_packet(&mut self) -> Result<Option<AudioPacket>, VorbisError> {
-        use self::lewton::audio::AudioReadError::AudioIsHeader;
```

Comment:
Why can you remove all this code? The function name suggests it's being used for seeking :-)

Is lewton even the decoder we're using?...

Reply:
It was a duplication left at the origin. You can see that seek and next_packet exist twice. By default, lewton is the decoder used unless you set differently your build command line (I don't know what it is you use)